uniffi_docs/src/lib.rs

```diff
@@ -144,3 +227,50 @@ pub fn extract_documentation(path: &Utf8Path) -> Result<Documentation> {
         impls,
     })
 }
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn it_parses_a_md_description() {
+        let description = std::fs::read_to_string("./tests/parsable_description.txt").unwrap();
```

Comment:
IMO, just inline that text in the test itself - it's not that long, and it's better to not have to jump around files to get the whole picture of the unit test (when it's possible).

You can use [indoc](https://docs.rs/indoc/latest/indoc/) for that (to be able to have pretty indent without affecting the input).